### PR TITLE
GUACAMOLE-682: docker build with optional maven profile

### PR DESCRIPTION
**Pull request by jolentes**
Extending docker build process to allow passing maven profile to include radius auth.

### CODE REVIEW

File: guacamole-docker/bin/build-guacamole.sh [hunk 1]
```diff
@@ -53,7 +54,12 @@ mkdir -p "$DESTINATION"
 #
 
 cd "$BUILD_DIR"
-mvn package
+
+if [ -z "$BUILD_PROFILE" ]; then
+  mvn package
+else
+  mvn -P "$BUILD_PROFILE" package
+fi
```
Comment (necouchman) [inline guacamole-docker/bin/build-guacamole.sh]: Please stick with accepted style of four-space indentations.

File: guacamole-docker/bin/build-guacamole.sh [hunk 2]
```diff
@@ -107,3 +113,11 @@ tar -xzf extensions/guacamole-auth-ldap/target/*.tar.gz \
     "*.jar"                                             \
     "*.ldif"
 
+#
+# Copy Radius auth extension if it was build
+#
+
+if [ -f extensions/guacamole-auth-radius/target/guacamole-auth-radius*.jar ]; then
+  mkdir -p "$DESTINATION/radius"
+  cp extensions/guacamole-auth-radius/target/guacamole-auth-radius*.jar "$DESTINATION/radius"
```
Comment (necouchman) [inline guacamole-docker/bin/build-guacamole.sh]: As with above, four-space indentations, please

File: guacamole-docker/bin/build-guacamole.sh (hunk 1 above)
Comment (jolentes) [reply guacamole-docker/bin/build-guacamole.sh]: corrected

File: guacamole-docker/bin/build-guacamole.sh (hunk 2 above)
Comment (jolentes) [reply guacamole-docker/bin/build-guacamole.sh]: corrected

File: guacamole-docker/bin/build-guacamole.sh [hunk 3]
```diff
@@ -41,6 +41,7 @@
 
 BUILD_DIR="$1"
 DESTINATION="$2"
+BUILD_PROFILE="$3"
```
Comment (mike-jumper) [inline guacamole-docker/bin/build-guacamole.sh]: This new `BUILD_PROFILE` parameter for `build-guacamole.sh` needs to be documented with an `@param` in the corresponding comment block (see the Doxygen-style block above covering this script and the other two parameters).

Comment (jolentes) [reply guacamole-docker/bin/build-guacamole.sh]: added the parameter documentation as requested

**Pull request merged**